pages/linux/kjv.md

```diff
@@ -0,0 +1,36 @@
+# kjv
+
+> The word of God available right on your desktop.
+> More information: <https://github.com/bontibon/kjv>.
+
+- Show all books:
+
+`kjv -l`
+
+- Open an individual book:
+
+`kjv Genesis`
+
+- Open an individual chapter of a book:
+
+`kjv Genesis 2`
+
+- Open individual verse(s) of a specific chapter of a book:
+
+`kjv John 3:16`
+
+- Open a range of verses in a book chapter:
+
+`kjv Proverbs 3:1-6`
+
+- Get a range of chapters and/or verses in a book:
```

Comment:
Your command example is about verses, not chapters. So, this example description is confused. Maybe this is better:
```suggestion
- Display a specific range of verses of a book from different chapters:
```